client/src/pages/Header/Header.tsx

```diff
@@ -57,8 +49,48 @@ const familiesTab = (
   </>
 );
 
-const HeaderPage = ({ history }: RouteComponentProps): JSX.Element => {
+const HeaderPage = ({ logoutApp }: HeaderPageProps): JSX.Element => {
+  const history = useHistory();
   const location = useLocation();
+
+  /**
+   * Logs out user from app
+   */
+  const logoutHandler = (): void => {
+    axios
+      .post(logoutPath)
+      .then(() => {
+        logoutApp();
+      })
+      .catch((err) => {
+        console.error(err);
+        notification.error({
```

Comment:
Kind of a meme to say logout failed?
Can we just delete the cookie and send the logout message to the backend without caring about the response?

Reply:
Gotta keep our user logged in forever lol. Good idea, I'll add that in and remove user notification.